API-source/models/teamgantt.php

```diff
@@ -19,43 +21,115 @@ public function fetch_teamgantt_todolist_id($logged_in_user) {
       }
     }
 
-    public static function fetch_list($logged_in_user, $query) {
-      if($query == 'today') {
-        $teamgantt_tasks = API::run($logged_in_user, 'GET', 'https://api.teamgantt.com/v1/tasks?today');
+    public static function synced_tasks($logged_in_user) {
```

Comment:
New function to pull tasks synced from TG. We can look it up by teamgantt id on todolists that belong to the logged in user.